associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -207,14 +208,17 @@
   <macro name="container-title">
     <choose>
       <if type="paper-conference" match="any">
-        <text variable="container-title" text-case="uppercase"/>
+        <text variable="event" text-case="uppercase"/>
         <text variable="number" prefix=", " suffix=". , "/>
         <text macro="issued-year" prefix=" " suffix=", "/>
         <text variable="publisher-place" suffix=". "/>
         <choose>
-          <if match="any" variable="title-short">
+          <else-if match="any" variable="container-title">
```

Comment:
you can't start `<choose>` with `else-if`. It has to start with `if`

Reply:
thanks! corrected!